### PR TITLE
[v1.5rc][F14-B1] My Team Manager (MTM)

### PR DESCRIPTION
**Pull request by ahmadsyafiqruhazat**
@jordancjq
@codeeong
@lohtianwei

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,10 +1,9 @@
 = Address Book (Level 4)
```
Comment (cjianhui) [inline README.adoc]: Do remember to update this to your product name!

File: README.adoc [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
-https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&utm_medium=referral&utm_content=se-edu/addressbook-level4&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a[Codacy Badge]]
+https://travis-ci.org/CS2103JAN2018-F14-B1/main[image:https://travis-ci.org/CS2103JAN2018-F14-B1/main.svg?branch=master[Build Status]]
+https://ci.appveyor.com/project/lithiumlkid/main/branch/master[image:https://ci.appveyor.com/api/projects/status/2itvfqp2b8xnaqqn/branch/master?svg=true[Build status]]
+https://coveralls.io/github/CS2103JAN2018-F14-B1/main?branch=master[image:https://coveralls.io/repos/github/CS2103JAN2018-F14-B1/main/badge.svg?branch=master[Coverage Status]]
```
Comment (cjianhui) [inline README.adoc:6]: Good job on updating the links!

File: src/test/java/seedu/address/model/AddressBookTest.java [hunk 3]
```diff
@@ -68,6 +77,37 @@ public void getTagList_modifyList_throwsUnsupportedOperationException() {
         addressBook.getTagList().remove(0);
     }
 
+    @Test
+    public void updatePerson_detailsChanged_personsAndTagsListUpdated() throws Exception {
```
Comment (cjianhui) [inline src/test/java/seedu/address/model/AddressBookTest.java:97]: Good job on using the three part naming convention for test methods as mentioned in our [coding standards](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#naming-conventions).

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -864,6 +1013,15 @@ Windows, Linux, Unix, OS-X
 [[private-contact-detail]] Private contact detail::
 A contact detail that is not meant to be shared with others
 
+[[ccm]] Codee's Championship Manager (CCM)::
```
Comment (cjianhui) [inline docs/DeveloperGuide.adoc]: Just to clarify, is your application's name Codee's Championship Manager (CCM) or My Team Manager (MTM)?

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -782,13 +782,14 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
...
 
+* football team managers
 * has a need to manage a significant number of contacts
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: Exclusive application for management of footballers and football teams that provides an enhanced listing of footballers and convenient lookup on updated information of players.
```
Comment (cjianhui) [inline docs/DeveloperGuide.adoc]: Good job on defining a clear and specific target user profile!

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -806,54 +808,201 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |password login |If user wants to protect certain information
+
+|`* *` |user |submit feedback to developers |developers can improve the application constantly
+
+|`*` |user |edit player’s name |
+
+|`*` |user with many persons in the address book |sort by player name |locate a person easily
+
+|`*` |user |sort by team name |identify group lineup easily
```
Comment (cjianhui) [inline docs/DeveloperGuide.adoc]: Good job on coming up with a variety of user stories that address the needs of your target user. To better reflect your target user, do update the `user` in your user stories to your target user profile, which is in this case a `team manager`.

**Review by cjianhui — COMMENTED** [review]
@lithiumlkid @codeeong @jordancjq @lohtianwei 
Good job on the PR! Just some minor comments. Also, remember to update your README, User Guide and Developer Guide to reflect your product name/ respective product descriptions as well.
Do leave this PR open as we will be tracking the milestones through this.

File: docs/DeveloperGuide.adoc (hunk 4 above)
Comment (jordancjq) [reply docs/DeveloperGuide.adoc]: Our application name will be My Team Manager (MTM), we have updated accordingly.

File: docs/DeveloperGuide.adoc [hunk 7]
```diff
@@ -798,62 +867,210 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+** 4a1. MTM inform user that it is an invalid add command.
++
+Use case ends.
+
+[discrete]
+=== Use case: Edit player's team
+
+*MSS*
+
+1.  User enter command to find
```
Comment (cjianhui) [inline docs/DeveloperGuide.adoc:1867]: For your MSS, do try to ensure it's more of like a back and forth interaction between the system and the user. So try to model and present your use cases in such a way that the system responds right after a user does something and so on and so forth.
In this case, you can write something like:
1. User enters command to find player's name.
2. MTM displays the list of players found.
3. ...

**Review by cjianhui — COMMENTED** [review]
@lithiumlkid @jordancjq @codeeong @lohtianwei 
Good job on updating the About Us, README and User Guide. Do keep the PR open after reading my remarks.
Ideally, you should think through the product's end behavior and write the entire UG, for all the commands which you would want to implement. Then mark the features as future (v2.0) which you do not intend to implement in the project duration, which I see you guys have done. Good job on that!

Finally, well done on updating your issues tracker reflect the user stories and tasks that you want to implement/complete during the duration of this project.
However, do remember to assign them to whoever is responsible for the task and also place them under the milestones you have created. 
You may refer to nus-cs2103-AY1718S2/forum#28 and https://github.com/MyTAOrg/DemoGradle/issues to see how you can go about doing it. I will talk more about it during tutorial as well.

**Review by cjianhui — COMMENTED** [review]
@lithiumlkid @jordancjq @codeeong @lohtianwei 
Good job on updating the DG to reflect your implementations. It also looks like you guys are on track to complete your product! However, do remember to supplement your implementation description with diagrams like sequence or class diagrams. 

Also, do remember to change your use cases to reflect a back and forth interaction between the user and the system as per mentioned during the last review.

Lastly, please remember to tag your repository as `v1.2` under Releases -> Tags. For the upcoming weeks you have to tag your repository to reflect the change in versions as well. Otherwise, keep up the good work!

**Pull request closed without merging**